### PR TITLE
Added endpoint

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added endpoint to get accommodation email list

### CODE REVIEW

File: HappyTravel.Nakijin.Api/Controllers/AccommodationsController.cs [hunk 1]
```diff
@@ -95,6 +95,25 @@ public async Task<IActionResult> Get([FromBody] List<string> accommodationHtIds)
             return Ok(accommodations);
         }
 
+        /// <summary>
+        /// Gets accommodation emails
+        /// </summary>
+        /// <param name="supplier">Supplier</param>
+        /// <param name="supplierAccommodationCode">Supplier Accommodation code </param>
+        /// <returns>Accommodation details</returns>
+        [HttpGet("suppliers/{supplier}/accommodations/{supplierAccommodationCode}/emails")]
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Controllers/AccommodationsController.cs:104]: I suggest `email-addresses`

Comment (hasmikharutyunyan) [reply HappyTravel.Nakijin.Api/Controllers/AccommodationsController.cs:104]: I think better to have one api for all contact information, not only for emails

**Comment by hasmikharutyunyan** [comment]
I suggest to wait for updater merge and after that impelemnt this api, to avoid conflicts

File: HappyTravel.Nakijin.Api/Controllers/AccommodationsController.cs (hunk 1 above)
Comment (AlexeyShok) [reply HappyTravel.Nakijin.Api/Controllers/AccommodationsController.cs]: Fixed

**Review by DFSko — APPROVED** [review]

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**